src/remote-config/remote-config-api.ts

```diff
@@ -56,10 +56,10 @@ export interface RemoteConfigCondition {
 
 /**
  * Represents a Remote Config condition in the dataplane.
- * A condition targets a specific group of users. A list of these conditions make up
- * part of a Remote Config template.
+ * A condition targets a specific group of users. A list of these conditions
+ * make up part of a Remote Config template.
  */
-export interface RemoteConfigServerCondition {
+export interface RemoteConfigServerNamedCondition {
```

Comment:
@lahirumaramba I see some interfaces with the RemoteConfig prefix and others without. Is there a guideline for what needs the prefix?

Reply:
I can't recall a reason. Looking at this proposal ([go/admin-sdk-remote-config](http://go/admin-sdk-remote-config)) from 2020 it looks like the types that were added later are missing the prefix.... maybe we weren't strict about the prefix in follow up proposals?

Reply:
I was just reviewing a change for the REST API and I think what happened is the TS API naming biased toward the REST API. The REST API has nested protos, but TS interfaces can't be nested. The parent protos had more qualified names than the child protos. When we created interfaces to match the protos, we retained the existing names.

I think the guideline we discussed against product prefixes still holds and will help the TS API be more consistent.